spawner/requirements.txt

```
@@ -1,4 +1,4 @@
-azure-common==1.1.8
-azure-mgmt-resource>=1.1.0
-azure-mgmt-containerinstance==0.1.0
-azure.servicebus
\ No newline at end of file
+azure-common==1.1.27
+azure-mgmt-resource==20.0.0
+azure-mgmt-containerinstance==9.1.0
+azure-servicebus==7.3.4
```

Does it have to be pinned to a specific version like 7.3.4 for feature/bug fix reason?
In general, I prefer not pinned to specific version as future version will come with bug fixes/new features.
```suggestion
azure-servicebus>=7.3.4
```

---

@yunhaoling I have updated by your comments.